### ISSUE TITLE
Failed to find kernel for Softmax(11) (node Softmax_0). Op with name (Softmax_0) and type (Softmax) kernel not found in CPUExecutionProvider

### ISSUE THREAD

**Issue by nsphuoc10** [microsoft/onnxruntime#9408]
Hi,
I followed the practices described in to build onnxruntime for all 4 targets (in android) https://onnxruntime.ai/docs/build/android-ios.html
   (1) converted onnx to ort files with basic optimization via --optimization_level basic & another ort files allowing onnx optimization in ort. 
   (2) Tried Release build with  ./build.sh --android --android_sdk_path /Library/Android/sdk/ --android_ndk_path /Library/Android/sdk/ndk/21.1.6352462 --android_api 29 --minimal_build extended --use_nnapi --disable_exceptions --build_shared_lib --skip_tests --include_ops_by_config ../required_operators.config --build_java  operators config file  was generated from step 1  
   (3) My config files contain 
      ai.onnx;1;Conv,ConvTranspose,GlobalAveragePool,Shape,Transpose
      ai.onnx;4;Concat
      ai.onnx;5;Reshape
      ai.onnx;6;Clip,HardSigmoid,Relu,Sigmoid,Tile
      ai.onnx;7;Add,Div,LSTM,Mul
      ai.onnx;9;BatchNormalization,MatMul
      ai.onnx;10;Resize
      ai.onnx;11;Clip,Concat,Conv,MaxPool,Slice,Softmax,Squeeze 

I am running on Api 29 on arm64-v8a but getting exception on ort: one with basic optimization
 The error is: “E/libc++abi: terminating with uncaught exception of type Ort::Exception: Failed to find kernel for Softmax(11) (node Softmax_0). Op with name (Softmax_0) and type (Softmax) kernel not found in CPUExecutionProvider. No matching hash…”

First time I built with config below:
      ai.onnx;1;GlobalAveragePool,Shape,Transpose
      ai.onnx;5;Reshape
      ai.onnx;6;HardSigmoid,Relu,Tile
      ai.onnx;7;Add,Div,LSTM,Mul
      ai.onnx;9;MatMul
      ai.onnx;11;Clip,Concat,Conv,MaxPool,Slice,Softmax,Squeeze

And the error is “Failed to find kernel for Sigmoid(6)….”
After that I updated with config at (3), and now it can’t find kernel for Softmax. I cleaned Release folder many times to rebuild but it’s no use. I don’t know why…

Please help me, thanks.

**Comment by nsphuoc10** [comment]
Thank you, I cloned, run it again. Now it works.
